src/Apps/Order/Routes/Shipping2/FulfillmentDetails.tsx

```diff
@@ -164,16 +185,16 @@ export const FulfillmentDetails: FC<FulfillmentDetailsFormProps> = ({
       onSubmit={handleSubmit}
     >
       <FulfillmentDetailsFormLayout
-        onSubmit={props.onSubmit}
         renderMissingShippingQuotesError={
           props.renderMissingShippingQuotesError
         }
         order={props.order}
+        onSubmit={props.onSubmit}
```

Comment:
is there any reason this can't be called from `handleSubmit`? Having two divergent `onSubmit` callbacks defined (one in formik props, one here)  is a bit confusing and i can imagine lead to subtle race conditions later. Can we roll this up?

Reply:
yes i agree and don't like this. it previously seemed impossible but let me take another look.